bool motors_killed
bool sensors_killed
